lib/octicons_react/src/index.d.ts

```diff
@@ -1,14 +1,19 @@
+// eslint-disable-next-line import/no-namespace
 import * as React from 'react'
 
-import {Icon} from './__generated__/icons'
+import { Icon } from './__generated__/icons'
 
 type Size = 'small' | 'medium' | 'large'
 
 export interface OcticonProps {
```

Comment:
Would it makes sense to extend `React.SVGAttributes<SVGElement>` here based on: https://github.com/primer/octicons/issues/907#issuecomment-1406793154 to support additional props for the `<svg>` element?

Reply:
Added the ability for it to be an svg 👍